e2e/upgrade-rbd.go

```diff
@@ -185,7 +185,7 @@ var _ = Describe("RBD Upgrade Testing", func() {
 				}
 				// fetch the path where volume is mounted.
 				mountPath := app.Spec.Containers[0].VolumeMounts[0].MountPath
-				filePath := filepath.Join(mountPath, "testClone")
+				filePath := filepath.Join(mountPath, "test")
```

Comment:
Will `filepath` be required if we are removing it's usage and calculating it in checksum function?

Reply:
Yes,  execCommandInPod() is using a filepath and hence I have not removed this. Though I have removed the later ones which didn't had any use.